packages/mui-base/src/Dialog/Backdrop/DialogBackdrop.tsx

```diff
@@ -24,6 +24,7 @@ const customStyleHookMapping: CustomStyleHookMapping<DialogBackdrop.OwnerState>
 };
 
 /**
+ * An overlay displayed beneath the popup. Renders a `<div>` element.
```

Comment:
Do we like this and the other descriptions I added in the PR @colmtuite @samuelsycamore?

I plan to add/rework the descriptions for all the components in this style. (Most are missing).

Reply:
I left a note on one but in general I'm happy with the descriptions and their style. 🤝